README.md

```diff
@@ -75,488 +79,579 @@ graph TD
+        "Amt": {
+          "Amt": 307.14,
+          "Ccy": "USD"
+        },
+        "Agt": {
+          "FinInstnId": {
+            "ClrSysMmbId": {
+              "MmbId": "dfsp001"
+            }
+          }
+        }
+      },
+      "InitgPty": {
+        "Nm": "April Blake Grant",
+        "Id": {
+          "PrvtId": {
+            "DtAndPlcOfBirth": {
+              "BirthDt": "1968-02-01",
+              "CityOfBirth": "Unknown",
+              "CtryOfBirth": "ZZ"
+            },
+            "Othr": {
+              "Id": "+27730975224",
+              "SchmeNm": {
+                "Prtry": "MSISDN"
+              }
+            }
+          }
+        },
+        "CtctDtls": {
+          "MobNb": "+27-730975224"
+        }
+      },
+      "Dbtr": {
+        "Nm": "April Blake Grant",
+        "Id": {
+          "PrvtId": {
+            "DtAndPlcOfBirth": {
+              "BirthDt": "1968-02-01",
+              "CityOfBirth": "Unknown",
+              "CtryOfBirth": "ZZ"
+            },
+            "Othr": {
+              "Id": "+27730975224",
+              "SchmeNm": {
+                "Prtry": "MSISDN"
+              }
+            }
+          }
+        },
+        "CtctDtls": {
+          "MobNb": "+27-730975224"
+        }
+      },
+      "DbtrAcct": {
+        "Id": {
+          "Othr": {
+            "Id": "+27730975224",
+            "SchmeNm": {
+              "Prtry": "MSISDN"
+            }
+          }
+        },
+        "Nm": "April Grant"
+      },
+      "DbtrAgt": {
+        "FinInstnId": {
+          "ClrSysMmbId": {
+            "MmbId": "dfsp001"
+          }
+        }
+      },
+      "CdtrAgt": {
+        "FinInstnId": {
+          "ClrSysMmbId": {
+            "MmbId": "dfsp002"
+          }
+        }
+      },
+      "Cdtr": {
+        "Nm": "Felicia Easton Quill",
+        "Id": {
+          "PrvtId": {
+            "DtAndPlcOfBirth": {
+              "BirthDt": "1935-05-08",
+              "CityOfBirth": "Unknown",
+              "CtryOfBirth": "ZZ"
+            },
+            "Othr": {
+              "Id": "+27707650428",
+              "SchmeNm": {
+                "Prtry": "MSISDN"
+              }
+            }
+          }
+        },
+        "CtctDtls": {
+          "MobNb": "+27-707650428"
+        }
+      },
+      "CdtrAcct": {
+        "Id": {
+          "Othr": {
+            "Id": "+27707650428",
+            "SchmeNm": {
+              "Prtry": "MSISDN"
+            }
+          }
+        },
+        "Nm": "Felicia Quill"
+      },
+      "Purp": {
+        "Cd": "MP2P"
+      }
+    },
+    "RgltryRptg": {
+      "Dtls": {
+        "Tp": "BALANCE OF PAYMENTS",
+        "Cd": "100"
+      }
+    },
+    "RmtInf": {
+      "Ustrd": "Payment of USD 30713.75 from April to Felicia"
+    },
+    "SplmtryData": {
+      "Envlp": {
+        "Doc": {
+          "Xprtn": "2021-11-30T10:38:56.000Z"
+        }
+      }
+    }
+  }
+}
+```
+
+**Expected Response for pacs008**
```

Comment:
Is this still correct? And also strangely, why is the pacs.008 message the only one with a response sample in the documentation?